src/app/static/src/js/components/runWorkflow/runWorkflowProgress.vue

```diff
@@ -196,6 +261,39 @@ export default Vue.extend<Data, Methods, unknown, Props>({
                         "An error occurred fetching workflow details";
                 });
         },
+        getWorkflowMetadata(){
+            api.get(`/workflows/${this.selectedWorkflowKey}/`)
+                .then(({data}) => {
+                    this.workflowMetadata = workflowRunDetailsToMetadata(data.data)
+                    this.getReportDependencies()
+                })
+                .catch((error) => {
+                    this.error = error;
+                    this.defaultMessage =
+                        "An error occurred fetching workflow details";
+                });
+        },
+        getReportDependencies() {
```

Comment:
This method repeats logic from the `runWorkflowSummary` component, although it is fairly heavily modified. Could put shared logic in utils method, mixin, or even the new vuex store?